src/System.Reflection.DispatchProxy/tests/project.json

```diff
@@ -20,7 +22,6 @@
     "netstandard1.3": {}
   },
   "supports": {
-    "coreFx.Test.netcore50": {},
```

Comment:
OK at a future point we may need to split these tests into 2 projects to enable us to run the majority on both .NET Core and .NET Native.